vector/src/main/java/im/vector/app/features/home/HomeActivityViewModel.kt

```diff
@@ -200,6 +205,11 @@ class HomeActivityViewModel @AssistedInject constructor(
                     }
                 }
                 .launchIn(viewModelScope)
+
+        if (session.hasAlreadySynced() && !hasCheckedBootstrap) {
+            hasCheckedBootstrap = true
```

Comment:
so, if I understand correctly, `hasCheckedBootstrap` is mostly to prevent any sequential call of `maybeVerifyOrBootstrapCrossSigning`? In this case couldn't we put it inside method, to ensure that no one will forget to set it when a new call of `maybeVerifyOrBootstrapCrossSigning` will be added in a future? Or this could cause multiple calls of this method since `withElementWellKnown` is async? 🤔

Reply:
If I'm not mistaken, `hasCheckedBootstrap` and previously `checkBootstrap` should check that verification or bootstrapping only happens once. I'll try to simplify it, though.